shared/lib_sandia.cpp

```diff
@@ -421,9 +426,9 @@ bool sandia_inverter_t::acpower(
 		if (B < 0.5 * Pso) B = 0.5 * Pso;
 		if (B > 2.0 * Pso) B = 2.0 * Pso;
 
-		Pac_each[m] = ((Paco / (A - B)) - C * (A - B)) * (Pdc[m] - B) + C0 * (Pdc[m] - B) * (Pdc[m] - B); //calculate Pac for this MPPT input and save it
-		PacNoPso_each[m] = ((Paco / A) - C * A) * Pdc[m] + C0 * Pdc[m] * Pdc[m]; //calculate Pac without operating losses (Pso = 0) for each MPPT input to store as Pso losses later
-		Pdc_total += Pdc[m];
+		Pac_each[m] = Pdc[m] / Pdc_total * (((Paco / (A - B)) - C * (A - B)) * (Pdc_total - B) + C * (Pdc_total - B) * (Pdc_total - B)); //calculate Pac for this MPPT input and save it
+		PacNoPso_each[m] = Pdc[m] * ((Paco / A) - C * A) + C * Pdc_total; //calculate Pac without operating losses (Pso = 0) for each MPPT input to store as Pso losses later
```

Comment:
@cwhanse @sjanzou I believe line 430 needs another `* Pdc_total` at the end of it to capture the squared power, otherwise these changes look right to me.

Reply:
@janinefreeman the full expression would be 

```
 Pdc[m] / Pdc_total * [ ((Paco / A) - C * A) * Pdc_total + C * Pdc_total * Pdc_total ]

```

I just canceled some factors.

Reply:
@cwhanse , @janinefreeman , based on the full expression (above) and dimensional analysis, I updated line 430 to include an additional set of parenthesis,   from:

		PacNoPso_each[m] = Pdc[m] * ((Paco / A) - C * A) + C * Pdc_total; // last addend is dimensionless , should be W

to:

		PacNoPso_each[m] = Pdc[m] * (((Paco / A) - C * A) + C * Pdc_total);